docs/agent/config/v2.mdx

```diff
@@ -1,43 +1,11 @@
 ---
-title: Configuration File
+title: "Version 2"
```

Comment:
this is "mostly"™️ the same as the original page with some fiddly changes

- pulled out global
- fixed some copy-text to be less confusing now that we have v2/v3
- fixed links (now that we have v2/v3)
- etc